### DISCUSSION TITLE
I Started a Blog

### DISCUSSION THREAD

**Discussion by giscus** [Lusito/blog#3] [Announcements]
# I Started a Blog

Recently, I wrote a rant about using the right tools for the job. This sparked some ideas and I needed a project to try it on. I had been writing posts from time to time, so it seemed like a good time to start my own blog."

https://blog.lusito.info/i-started-a-blog.html

**Comment by Lusito** [comment]
If you want comments on your (development-focused) webpage as well, you might like [giscus](https://giscus.app/). It was quite easy to set up.
